### DISCUSSION TITLE
Trivy Vulnerability Results Report

### DISCUSSION THREAD

**Discussion by rwlib3** [goharbor/harbor#18310] [Q&A] [answered]
Hello all! 

I need to get a report of all the projects, repos and artifacts with the number of vulnerabilities found. I'm trying to pull all the data together using the API, but not finding the specific vulnerability severity (critical, high, etc...) and count.

Any insight would be greatly appreciated!

**Comment by chlins** [comment, marked as answer]
Hi, if harbor version is 2.6 you can use CVE export feature to export the vulnerabilities as a CSV file for a project. FYI: https://goharbor.io/blog/harbor-2.6/
